packages/core/lib/setup-tests.js

```diff
@@ -0,0 +1,11 @@
+import {getRpcClient, initRpcClient} from './rpc-client';
```

Comment:
this feels incomplete?

Reply:
seems an old comment, didn't find any issue in the current code